### PR TITLE
Check that batch table supports arrays and objects

### PR DESCRIPTION
**Pull request by lilleyse**
For https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/65#issuecomment-187245679

Updated a test to check that the batch table can support arrays and objects.

### CODE REVIEW

File: Specs/Data/Cesium3DTiles/Batched/BatchedWithBatchTable/tileset.json [hunk 1]
```diff
@@ -18,6 +18,14 @@
...
       "minimum": 6,
       "maximum": 14
+    },
+    "info": {
+      "minimum": null,
+      "maximum": null
+    },
+    "rooms": {
+      "minimum": null,
+      "maximum": null
```
Comment (lilleyse) [inline Specs/Data/Cesium3DTiles/Batched/BatchedWithBatchTable/tileset.json]: Would would the spec expect these to be?

Comment (pjcozzi) [inline Specs/Data/Cesium3DTiles/Batched/BatchedWithBatchTable/tileset.json]: lol, I still have a TODO

Comment (pjcozzi) [inline Specs/Data/Cesium3DTiles/Batched/BatchedWithBatchTable/tileset.json]: I still have a TODO for metadata here for non-numeric fields.

File: Specs/Scene/Cesium3DTileBatchTableResourcesSpec.js [hunk 2]
```diff
@@ -326,6 +326,14 @@ defineSuite([
             // check that the 2nd resource has an id of 2
             expect(content.getFeature(2).getProperty('id')).toEqual(2);
 
+            // Check that a property can be an array
+            expect(content.getFeature(2).getProperty('rooms')).toEqual(['room2_a', 'room2_b', 'room2_c']);
```
Comment (pjcozzi) [inline Specs/Scene/Cesium3DTileBatchTableResourcesSpec.js]: Do any of these tests modify with `setProperty` and verify the results?  If not, can we add a new test?

**Comment by lilleyse** [comment]
Updated with new tests.

**Comment by pjcozzi** [comment]
Looks good, thanks!

**Pull request merged**